.github/workflows/test-go.yaml

```diff
@@ -84,3 +84,6 @@ jobs:
           path-to-profile: cover.out
           working-directory: ${{ env.GOPATH }}/src/github.com/kubeflow/trainer
           parallel: true
+          parallel-finished: true
```

Comment:
I think, we should also enable back this GitHub action on `push`, since we need to report status to Coveralls after the PR is merged, right @tenzen-y @Electronic-Waste @astefanutti ?
https://github.com/kubeflow/trainer/blob/master/.github/workflows/test-go.yaml#L4

Reply:
OK, I recover the `test-go.yml` and wait for push, maybe we need to run ci on `master` branch?